modifier.py

```diff
@@ -127,7 +134,7 @@ def modify_song(songJson, modifiers: list[Modifier]):
                 syllable['len'] = int(newTime)
         else:
             line['start'] = str(timedelta(seconds=pytimeparse.parse(line['start'])) + modifier.offset)
-            line['end'] = str(timedelta(seconds=pytimeparse.parse(line['end'])) + modifier.offset)
+            line['end'] = str(timedelta(seconds=pytimeparse.parse(line['end'])) + modifier.offset - modifier.trim)
```

Comment:
Or well, the even easier way to deal with this is to also trim the length of the last syllable here.

I think we can live with things potentially being weird if the timings are manually edited.

Reply:
Updated, the last syllable should be retimed appropriately now.